.github/workflows/ci.yml

```diff
@@ -0,0 +1,33 @@
+name: CI-Build
+
+on: 
+  push:
+  workflow_dispatch:
+  schedule:
+    - cron: '21 21 * * *'
+
+permissions:
+  contents: write
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Generate Monitor
+      run: |
+        pwd
+        ls -la
```

Comment:
`pwd` and `ls -la` needed? 🤔

Reply:
seems like he was on an investigation trip to find the build.sh!

Reply:
yes, exactly .. missed the prefix `./`and this is how I got there :)